tasks/main.yml

```diff
@@ -138,6 +138,15 @@
     src: "humio@.service.j2"
     dest: /etc/systemd/system/humio@.service
   notify: Restart Humio
+  when: not humio_disable_numactl
+
+- name: Create Humio SystemD template configuration
+  tags: humio-update
+  template:
+    src: "humio.service.j2"
```

Comment:
Wouldn't it be better just to make a small adjustment to the existing template which would leave out the `/usr/bin/numactl --cpunodebind=${NUMA_NODES} --membind=${NUMA_NODES}` part of `ExecStart` command?

Reply:
@SaaldjorMike I made the change you suggested. Have a look again please.